mmv1/third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -27,6 +28,12 @@ func resourceDataflowFlexTemplateJob() *schema.Resource {
 		Read:   resourceDataflowFlexTemplateJobRead,
 		Update: resourceDataflowFlexTemplateJobUpdate,
 		Delete: resourceDataflowFlexTemplateJobDelete,
+		CustomizeDiff: customdiff.All(
+			resourceDataflowJobTypeCustomizeDiff,
+		),
+		Importer: &schema.ResourceImporter{
```

Comment:
Was this resource previously not importable?

Reply:
nope